generated/meson.build

```diff
@@ -0,0 +1,55 @@
+
+sourcery = executable('sourcery', join_paths(meson.source_root(), 'src', 'sourcery.cpp'), native: true)
+
+make_html = executable('make_html', 'make_html.cpp', native: true)
+
+html_list = [
+    join_paths(meson.source_root(), 'lsp/header.html'),
```

Comment:
... or tbh, if this happens multiple times, you can define a files() list in one directory and reuse it in another directory to retain knowledge of the directory context.